### PR TITLE
Clarify Drupal now supports JSON:API in core as of 8.7.0.

### PR DESCRIPTION
**Pull request by Eli-TW**
See https://www.drupal.org/node/3041438

### CODE REVIEW

**Comment by dgeb** [comment]
@gabesullice this one is definitely in your department! Could you please review?

File: implementations/index.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ and writing of JSON:API documents.
...
+* [drupal](https://www.drupal.org/project/drupal) The Drupal CMS supports exposing all data it manages (entities) according to the JSON:API specification with no additional modules required as of 8.7.0. [`jsonapi_extras`](https://www.drupal.org/project/jsonapi_extras) is an optional extra module to change resource type names, field names and more. And [`openapi`](https://www.drupal.org/project/openapi) is another optional module, that is able to generate an OpenAPI/Swagger representation of the API provided by the `jsonapi` module with a ReDoc-powered UI. And [`jsonapi`](https://www.drupal.org/project/jsonapi) is a Drupal module that allows versions of Drupal 8 prior to 8.7.0 to expose data via JSON:API.
```
Comment (gabesullice) [inline implementations/index.md]: Thanks @Eli-TW!

I think it makes sense to include JSON:API Extras, but I think it's outside the scope of this listing to talk about specific version of Drupal core or the `openapi` module. Would you mind removing the version-dependent stuff and OpenAPI/Redoc language?

(I know you didn't add it yourself ;) )

**Review by gabesullice — CHANGES_REQUESTED** [review]

File: implementations/index.md (hunk 1 above)
Comment (Eli-TW) [reply implementations/index.md]: Done!

**Review by gabesullice — APPROVED** [review]

**Comment by gabesullice** [comment]
Thanks @Eli-TW!

**Pull request merged**